schema-dex-agg.graphql

```diff
@@ -0,0 +1,514 @@
+# Subgraph Schema: DEX AGG
+# Version: 1.0.0
+# See https://github.com/messari/subgraphs/blob/master/docs/Schema.md for details
```

Comment:
Do you think it is worth including here since it is the first schema? I guess I could include how it differs from the DEX schema. Its mostly a simplified version with the addition of Virtual Pools.

Reply:
Yeah I guess not, resolved.